setup.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+from setuptools import setup, find_packages
+
+VERSION_FILE = 'VERSION'
+
+
+def read_version():
+    with open(VERSION_FILE, 'r') as fp:
+        return tuple(map(int, fp.read().split('.')))
+
+
+def read(fname):
+    return open(os.path.join(os.path.dirname(__file__), fname)).read()
+
+
+setup(
+    name="cgp-bouncer",
+    description="The whitelist checker for authentication with CGP HCA Data Store",
```

Comment:
Please update the description to one similar to that proposed for the repo title.

Reply:
got it